src/main/java/sirius/kernel/nls/Babelfish.java

```diff
@@ -80,6 +80,24 @@
         return translationMap.values().stream().filter(e -> e.getKey().startsWith(key));
     }
 
+    /**
+     * Enumerates all translations which have not been accessed yet.
+     *
+     * @return a list of all unused translations.
+     */
+    public Stream<Translation> getUnusedTranslations() {
+        return translationMap.values().stream().filter(e -> !e.isUsed());
+    }
+
```

Comment:
Sollte man die nicht irgendwo anschauen können? via console oder so? Die Methode ist so ja erstmal unused, oder?

Reply:
Denke das kommt wenn dann in web oder biz